_episodes/03-create.md

```diff
@@ -273,6 +283,14 @@ $ rm -r thesis
 ~~~
 {: .bash}
 
+However, if the `thesis` directory is empty, we can use `rmdir thesis` to remove the directory
+
+~~~
+$ rm thesis/draft.txt
+$ rmdir thesis
+~~~
+{: .bash}
+
```

Comment:
-1. Let's not introduce a new command `rmdir` without an explanation for why it's preferable to `rm -r`